### PR TITLE
Fix in TGeoMCGeometry::Mixture(Int_t& kmat, const char* name, Float_t…

### PR DESCRIPTION
**Pull request by agheata**
Cherry picked from Ivana's commit f245cb4
Ivana needs this in the patches branch

### CODE REVIEW

**Comment by agheata** [comment]
Cherry-picked also the previous commit that Ivana wanted

File: montecarlo/vmc/inc/TMCParticleStatus.h [hunk 1]
```diff
@@ -48,7 +48,7 @@ struct TMCParticleStatus {
    /// Print all info at once
    void Print() const
    {
-      ::Info("Print", "Status of track");
```
Comment (Axel-Naumann) [inline montecarlo/vmc/inc/TMCParticleStatus.h]: This should not be reverted! Please make sure this wasn't reverted in master either!

**Comment by agheata** [comment]
Kept only the cherry-pick of d3e4e1b. It rebuilds now.
@ihrivnac can you check that this is what you wanted?

**Comment by ihrivnac** [comment]
Thank you, @agheata , I confirm.

**Comment by agheata** [comment]
@Axel-Naumann this is ready to be merged

**Pull request merged**